src/lib/utils/hooks/useGetInfoFromIds.ts

```diff
@@ -91,46 +125,13 @@ export default function useGetInfoFromIds<
   const idProp = (type: HookType) =>
     type === 'USER_PROFILE' ? 'ownerId' : 'id'
 
-  const storageKey = (type: HookType) =>
-    type === 'USER_PROFILE'
-      ? SynapseConstants.USER_PROFILE_STORAGE_KEY
-      : SynapseConstants.ENTITY_HEADER_STORAGE_KEY
-
   // look at current list of data, see if incoming ids has new data,
   // if so grab those ids
   const curList = data.map(el => el[idProp(type)])
   const incomingList = ids.filter(el => el !== SynapseConstants.VALUE_NOT_SET)
   const newValues = uniq(without(incomingList, ...curList))
 
-  const saveToSessionStorage = (data: T[], type: HookType) => {
-    if (!data.length) {
-      return
-    }
-    //get what's there
-    const dataInStorage = sessionStorage.getItem(storageKey(type))
-    try {
-      const dataInStorageAsObjectArr: T[] = dataInStorage
-        ? JSON.parse(dataInStorage)
-        : []
-      //get an array of ids for items already in storage
-      const ids = dataInStorageAsObjectArr.map(item => item[idProp(type)])
-      //push all the new data if ids are new
-      for (const dataObject of data) {
-        if (!ids.includes(dataObject[idProp(type)])) {
-          dataInStorageAsObjectArr.push(dataObject)
-        }
-      }
-      sessionStorage.setItem(
-        storageKey(type),
-        JSON.stringify(dataInStorageAsObjectArr),
-      )
-    } catch (e) {
-      sessionStorage.setItem(storageKey(type), JSON.stringify(data))
-    }
-  }
-
-  // Alina TODO: check if the items are already in Local Storage before making server call.
```

Comment:
Note Alina's comment.  She was right, local storage cache is not used, so there's no point in saving the results to web storage

Reply:
There is a point!  TotalQueryResults uses the cache to figure out the display name to be rendered.